src/subpages/account.tsx

```diff
@@ -151,112 +155,67 @@ const AccountPage: React.FC<AccountPageProps> = ({ location }) => {
-  const columns: IColumn[] = [
-    {
-      key: 'coulmnNonce',
-      name: 'Nonce',
-      fieldName: 'nonce',
-      minWidth: 5,
-      maxWidth: 50,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'string',
-      isPadded: true,
-    },
-    {
-      key: 'columnId',
-      name: 'ID',
-      fieldName: 'id',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-      // FIXME: We'd better to use absolute paths and make Gatsby automatically
-      // to rebase these absolute paths on the PATH_PREFIX configuration.
-      onRender: ({ id }: Transaction) => (
-        <Link href={`../transaction/?${id}`}>{id}</Link>
-      ),
-    },
-    {
-      key: 'columnSignature',
-      name: 'Signature',
-      fieldName: 'signature',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-    },
-    {
-      key: 'columnSigner',
-      name: 'Signer',
-      fieldName: 'signer',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-      onRender: ({ signer }: Transaction) => (
-        // FIXME: We'd better to use absolute paths and make Gatsby automatically
-        // to rebase these absolute paths on the PATH_PREFIX configuration.
-        <Link href={`./?${signer}`}>{signer}</Link>
-      ),
-    },
-    {
-      key: 'columnTimestamp',
-      name: 'Timestamp',
-      fieldName: 'timestamp',
-      minWidth: 100,
-      maxWidth: 200,
-      isRowHeader: true,
-      isResizable: true,
-      isSorted: false,
-      isSortedDescending: true,
-      data: 'number',
-      isPadded: true,
-      onRender: ({ timestamp }: Transaction) => (
-        <Timestamp timestamp={timestamp} />
-      ),
-    },
-  ];
+interface TransactionListWrapProps {
+  signed?: Transaction[];
+  involved?: Transaction[];
+  missingNonces?: number[];
+  loading: boolean;
+}
 
-  return (
-    <DetailsList
-      items={transactions}
-      columns={columns}
-      selectionMode={SelectionMode.none}
-      getKey={(tx: Transaction) => tx.id}
-      setKey="set"
-      layoutMode={DetailsListLayoutMode.justified}
-      isHeaderVisible={true}
-      // FIXME: We'd better to use absolute paths and make Gatsby automatically
-      // to rebase these absolute paths on the PATH_PREFIX configuration.
-      onItemInvoked={({ id }: Transaction) => navigate(`../transaction/?${id}`)}
+const TransactionListWrap: React.FC<TransactionListWrapProps> = ({
+  signed,
+  involved,
+  missingNonces,
+  loading,
+}) => (
+  <>
+    <h2>Signed Transactions {counter(signed)}</h2>
```

Comment:
> [In modern English-language printing, no space is placed before a colon and a single space is placed after it.][1]

[1]: https://en.wikipedia.org/wiki/Colon_(punctuation)#Spacing

```suggestion
    <h2>Signed Transactions{counter(signed)}</h2>
```